### ISSUE TITLE
User "system:serviceaccount:monitoring:kor" cannot list resource "acraccesstokens" in API group

### ISSUE THREAD

**Issue by Boniker** [yonahd/kor#386]
**Describe the bug**
logs from `kor-exporter`
```
collecting unused resources
Failed to get Crds: [acraccesstokens.generators.external-secrets.io](http://acraccesstokens.generators.external-secrets.io/) is forbidden: User "system:serviceaccount:monitoring:kor" cannot list resource "acraccesstokens" in API group "[generators.external-secrets.io](http://generators.external-secrets.io/)" at the cluster scope
```

**To Reproduce**
You need to have CRDs of external-secrets inside the cluster and integrate kor to k8s cluster as exporter

**Expected behavior**
info log without error
```
collecting unused resources
```

**OS version, architecture and kor version**
0.1.14

**Additional context**
It can be fixed in anyway:
- upgrade the helm chart with possibility to add extra resources in values.yaml file
- hardcode it [here](https://github.com/yonahd/kor/blob/050c06d898126b29f9fb9b6ea021d43f11bdbed1/charts/kor/templates/role.yaml#L11)

**Comment by yonahd** [comment]
Did the ClusterRole apply for you? 
https://github.com/yonahd/kor/blob/050c06d898126b29f9fb9b6ea021d43f11bdbed1/charts/kor/templates/role.yaml#L66

**Comment by Boniker** [comment]
ClusterRole successfully applied. No errors during helm install. 
An issue appears during kor-exporter run

**Comment by Boniker** [comment]
@yonahd Kor has an option to delete unused resources, but the necessary permissions are not granted in the Role/ClusterRole configurations in the Helm chart

**Comment by yonahd** [comment]
@Boniker correct. This is by design as we do not want to encourage unsupervised resource cleanups
